moto/state_connection.py

```diff
@@ -96,11 +99,10 @@ def _worker(self) -> None:
                 with self._lock:
                     self._robot_status = deepcopy(msg.body)
 
-            if not self._initial_response.is_set() and \
-                (self.robot_status is not None and \
-                self._joint_feedback is not None and \
-                self.joint_feedback_ex is not None):
+            if not self._initial_response.is_set() and (
+                self.robot_status is not None
+                and self._joint_feedback is not None
+                and self.joint_feedback_ex is not None
+            ):
                 print("At least one message of every time has been received.")
```

Comment:
Might be we should move to use logging and not prints.

Reply:
Removing the print statement now. 

I think logging should be implemented everywhere, so I added it to the project board as a to-do. 
[Link](https://github.com/ra-mtp-ntnu/moto/projects/1#card-74143509)